src/Gir/Generation/IGeneratableExtensions.cs

```diff
@@ -71,7 +71,9 @@ public static void GenerateCallableDefinition(this ICallable callable, IndentWri
 			var returnType = callable.GetReturnCSharpType(writer);
 
 			// generate ReturnValue then Parameters
-			writer.Write(string.Format("{0} {1} {2}", returnType, callable.Name.ToCSharp (), "PARAMS"));
+			// FIXME, probably don't need the instance parameters?
```

Comment:
We can get the type from the symbol table, since Array exposes a CType. We can probably make `Array` an ISymbol and implement it.